pkg/config/config.go

```diff
@@ -25,21 +25,18 @@ const (
 	defaultCurrentContext = "relaysh"
 )
 
-var (
-	defaultAPIURL = &url.URL{Scheme: "https", Host: "api.relay.sh"}
-	defaultUIURL  = &url.URL{Scheme: "https", Host: "app.relay.sh"}
-	defaultWebURL = &url.URL{Scheme: "https", Host: "relay.sh"}
-)
-
 var defaultContexts = map[string]APIContext{
 	"relaysh": {
 		Name:      "relaysh",
-		APIDomain: defaultAPIURL,
-		UIDomain:  defaultUIURL,
-		WebDomain: defaultWebURL,
+		APIDomain: &url.URL{Scheme: "https", Host: "api.relay.sh"},
+		UIDomain:  &url.URL{Scheme: "https", Host: "app.relay.sh"},
+		WebDomain: &url.URL{Scheme: "https", Host: "relay.sh"},
 	},
 	"dev": {
-		Name: "dev",
+		Name:      "dev",
+		APIDomain: &url.URL{Scheme: "http", Host: "relay-api.local:8080"},
+		UIDomain:  &url.URL{Scheme: "http", Host: "relay-ui.local:8080"},
+		WebDomain: &url.URL{Scheme: "http", Host: "relay-ui.local:8080"},
```

Comment:
Also just a side note that this is supposed to be to link into the marketing/docs website, not the app.

Reply:
Yeah, this is just a placeholder until we have something in place for the local environment. The `WebDomain` does not appear to be used anywhere right now. I'd rather not over-complicate that logic right now.